internal/testutil/test_hcls.go

```diff
@@ -239,38 +244,43 @@ func RHclCloudRouterConnectionAws() RHclCloudRouterConnectionAwsResult {
 			ResourceName: resourceName,
 		},
 		AwsAccountID: os.Getenv(PF_CRC_AWS_ACCOUNT_ID_KEY),
+		AccountUuid:  os.Getenv(PF_ACCOUNT_ID_KEY),
+		Speed:        CloudRouterConnAwsSpeed,
 		Pop:          pop,
+		CloudRouter:  hclCloudRouterRes,
 	}
 }
 
 // packetfabric_cloud_router_bgp_session
 func RHclBgpSession() RHclBgpSessionResult {
 
-	hclCloudRouterRes := RHclCloudRouter()
 	hclCloudConnRes := RHclCloudRouterConnectionAws()
```

Comment:
to replace with `RHclCloudRouterConnectionPort()` as it won't be possible to setup BGP without accepting the AWS connection first.